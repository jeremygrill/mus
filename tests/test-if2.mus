Int main(){
	
	Int i;
	Note n;
	Chord c;

	i = 1;

	while(i < 20){
		if(true){
			n = (60+i,10|10);
			c = c * n;
		}

		i = i * 2;
	}

	printc(c);
	
	
	return 0;
}
