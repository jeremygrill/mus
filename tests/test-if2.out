[(61,10|10)(62,10|10)(64,10|10)(68,10|10)(76,10|10)]
